frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.sass

```diff
@@ -3,3 +3,9 @@ ng-select.wp-inline-create--reference-autocompleter
 
   .ng-clear-wrapper
     display: none
+    
+.ng-select.ng-select-single .ng-select-container .ng-value-container .ng-input 
+  position: inherit
+
+.ng-dropdown-panel.ng-select-bottom 
+  dispalay: inline-table
```

Comment:
There are two issues with this line: 
First, there is a typo in the rule name (disp **a** lay). So I assume that this rule is never applied, and can thus be removed. 
Second, as a more general notice: Please use all kinds of `display: table, display: inline-table, display: table-cell, etc` only when you are actually in a table. Everything else is quite hacky and will lead to unexpected side effects sooner or later.